crates/snapshot/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+mod error;
+mod snapshotter;
+
+pub use error::SnapshotterError;
+pub use snapshotter::{SnapshotTargets, Snapshotter, SnapshotterResult, SnapshotterWithResult};
```

Comment:
needs regular reth crate headers and high level doc

ref 

https://github.com/paradigmxyz/reth/blob/2138a8b587ca7dc985a23b20d346dbf6b60acc35/crates/consensus/beacon/src/lib.rs#L3-L10

Reply:
fixed